### DISCUSSION TITLE
[Application] IPFS SOUKO

### DISCUSSION THREAD

**Discussion by noritaka88ta** [filecoin-project/community#147] [Project Showcase]
News release about Joint research
https://nonentropy.jp/news/nonentropy-japan-to-begin-joint-research-with-graduate-school-of-information-science-and-technology-osaka-university/

News release about IPFS SOUKO
https://nonentropy.jp/news/ipfs-souko/

IPFS SOUKO is a digital archive service that enables smooth uploading and downloading of data between conventional storage (including cloud) and IPFS/Filecoin distributed storage.

IPFS and Filecoin are cutting-edge technologies, and there are still very few people who are proficient in handling them. The simple UI/UX of IPFS SOUKO allows anyone to store (upload) or retrieve (download) their data to IPFS or Filecoin distributed storage.

NonEntropy Japan Launches Joint Research with Osaka University on
* The Optimization of the IPFS/Filecoin network performance, and labor-saving and data allocation in the distributed network.
* Security Enhancement to Promote IPFS/Filecoin Distributed Storage

![IPFS SOUKO image](https://gyazo.com/748f7b883779a2163332d8442a73e662.jpeg)
![IPFS SOUKO structure](https://gyazo.com/fda07acd1415e71eed26b429510f6516.jpeg)

**Comment by daichi000** [comment]
## Updates as the end of May 7

<img src="https://user-images.githubusercontent.com/35623457/117596951-7c576f80-b17f-11eb-9a07-e944641db735.png" width="800">

- Added the ability to upload files to IPFS
- Drag-and-drop uploading is now possible
- A security check animation runs during upload
- Added the ability to download and delete files in the detail screen

**Comment by Maaaaru** [comment]
## Updates as the end of May 14
![スクリーンショット 2021-05-17 10 45 29](https://user-images.githubusercontent.com/50836297/118421930-1f1e6980-b6fd-11eb-8826-703c3b3e8aca.png)


- Added support for mobile phones
- Fixed a bug that delayed CSS reflection
- URLs are now displayed in browser tabs
- Fixed file misalignment
- Other minor bugs fixed

**Comment by Igarashi-Akira** [comment]
## Updates as the end of June 6
![image (2)](https://user-images.githubusercontent.com/56008175/120947653-20cad280-c77b-11eb-931b-5947473a08aa.png)


- added FilData (https://www.fildata.info) Link
FilData is Filecoin related data dashboard.

**Comment by daichi000** [comment]
## New Release

### Next-generation distributed data storage service "IPFS SOUKO" and Terada Warehouse start digital data archiving demonstration experiment

Ltd. (Head office: Chiyoda-ku, Tokyo, President: Takuo Nishimura, hereinafter referred to as "the Company"), a distributed storage business that plays an important role in the infrastructure of the Web 3.0 society under the slogan "digitally preserving everything in the world," has collaborated with Terada Warehouse Co. (President and CEO: Kohei Terada, Shinagawa-ku, Tokyo; hereinafter referred to as "Terada Soko") to archive digital data, including cultural assets and art-related data, through "IPFS SOUKO," which realizes secure digital archiving through IPFS and Filecoin distributed storage. (President: Kohei Terada, Shinagawa-ku, Tokyo; hereinafter referred to as "Terada Soko") to start a demonstration experiment to archive digital data, including cultural properties and art-related data.

![image](https://user-images.githubusercontent.com/35623457/127427917-194ad39d-7c31-4f4c-ae43-151d5ed334f1.png)

- Features of IPFS SOUKO
IPFS SOUKO is a digital archive service that enables smooth uploading and downloading of data between conventional storage (including cloud) and IPFS and Filecoin distributed storage. With its simple UI and UX, even those who are not familiar with IPFS and Filecoin, the most advanced technologies, can easily store (upload) and retrieve (download) their data to and from the IPFS and Filecoin distributed storage. We are pleased to announce that we have started to accept cultural assets and art-related digital data for archiving.

- Significance of archiving cultural assets and art-related digital data at IPFS SOUKO
Passing on cultural assets and works of art to future generations is an extremely important part of protecting human cultural assets. By reducing the need for access to the original materials, the digital data will minimize physical damage to the original materials in the future, and can be transferred to remote locations via digital communication or used as material for new content. It is also possible to transfer them to remote locations via digital communication and use them as materials for new content. In addition to cultural assets and works of art, images that record scenes and customs of the past are considered to be a valuable cultural heritage for a country or region, and it is important to protect and preserve them from the danger of loss and disappearance. With the recent improvement of computer processing technology, it has become possible to construct high-precision 3D graphics and convert them to AR, and the data capacity is rapidly increasing. IPFS/Filecoin distributed storage is a platform-independent storage method that enables secure data archiving in the future, and is suitable for the preservation of such cultural assets and art-related digital data. and art-related digital data.


More information about IPFS SOUKO can be found at
https://nonentropy.jp/about-services/
